### PR TITLE
Remove less important docs build from CI

### PR DESCRIPTION
**Pull request by jenshnielsen**
* Drop 3.7 on ubuntu (we are already building with 3.7 on window)
* Drop 3.8 (we both test with 3.7 and 3.9 so this is very unlikely to break)

### CODE REVIEW

File: .github/workflows/docs.yaml [hunk 1]
```diff
@@ -18,18 +18,20 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest]
-        python-version: [3.7, 3.8, 3.9, "3.10"]
+        python-version: [3.7, 3.9, "3.10"]
         exclude:
           - os: windows-latest
             python-version: 3.8
```
Comment (astafan8) [inline .github/workflows/docs.yaml:24]:
```suggestion
```

**Review by astafan8 — APPROVED** [review]

**Comment by jenshnielsen** [comment]
bors merge

**Comment by jenshnielsen** [comment]
bors merge

**Comment by jenshnielsen** [comment]
Bors merge

**Pull request merged**